docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -146,6 +156,26 @@ pulp_python, and so forth. Any plugins in folders that start with ``pulp_`` that
 in your host machine's code folder alongside the Pulp platform repository should have been installed
 and configured for virtualenv.
 
+Using Vagrant
+-------------
+
+Vagrant enviroment by default provides some useful built-in commands. More information about them
+can be found in command ``phelp``.
+
+List of most useful commands:
+
+* ``pstart`` - Starts all pulp related servicies
+* ``ppopulate`` - Load default testing repositories
+
+.. note::
+
+    You have to issue ``pstart`` after starting vagrant.
+
+.. note::
+
+    It can hapen that mongo won't start due to the incorrectly stopped vagrant.
```

Comment:
This can be rephrased to:

"If Vagrant is stopped incorrectly, mongo may not be able to start."